terraform/aws/analytical-platform-data-production/airflow/terraform.tfvars

```diff
@@ -47,10 +47,17 @@ prod_vpc_sg_name = "airflow-prod"
 # EKS Cluster
 ##################################################
 
-dev_eks_role_arn    = "arn:aws:iam::593291632749:role/airflow-dev-eksRole-role-211908c"
-dev_cluster_sg_name = "airflow-dev-eksClusterSecurityGroup-6a4dde4"
-dev_node_sg_id      = "sg-01930457ae391c7f0"
-
-prod_eks_role_arn    = "arn:aws:iam::593291632749:role/airflow-prod-eksRole-role-de6b4f5"
-prod_cluster_sg_name = "airflow-prod-eksClusterSecurityGroup-6ab84a6"
-prod_node_sg_id      = "sg-0f73e78564012634a"
+dev_eks_cluster_name           = "airflow-dev"
+dev_eks_role_arn               = "arn:aws:iam::593291632749:role/airflow-dev-eksRole-role-211908c"
+dev_cluster_additional_sg_name = "airflow-dev-eksClusterSecurityGroup-6a4dde4"
+dev_cluster_additional_sg_id   = "sg-0bcd3cf5dc6d7b314"
+dev_cluster_node_sg_id         = "sg-01930457ae391c7f0"
+dev_cluster_node_sg_name       = "airflow-dev-nodeSecurityGroup-f3e4ecd"
```

Comment:
How many of these need hardcoding versus can be retrieved as attributes post eks cluster import?

Reply:
All but the `<env>_cluster_name` need to be hardcoded like this. Given this we feel it is more appropriate to keep them.

If we want to reference the clusters as data blocks this adds two new data block resources with potentially ugly names - due to 'dev' being deployed in the production account. e.g.

``` terraform
data "aws_eks_cluster" "analytical_platform_data_production_dev" {
  provider = aws.analytical-platform-data-production

  name = "airflow-dev"
}

data "aws_eks_cluster" "analytical_platform_data_production_prod" {
  provider = aws.analytical-platform-data-production

  name = "airflow-prod"
}
```

Ultimately this would lead to us updating the line name line in our `aws_eks_cluster` resource blocks from:
``` terraform
name     = var.dev_eks_cluster_name
```
to:
``` terraform
name     = data.aws_eks_cluster.analytical_platform_data_production_dev.id
```

This doesn't feel like win since we are:
- moving one variable away from comparable variables into a data block (it still needs to be defined in there)
- making data blocks with less than appealing names